### ISSUE TITLE
b3nl and bk in CCL

### ISSUE THREAD

**Issue by jablazek** [LSSTDESC/CCL#900]
Add the interface with fastpt for b3nl and bk bias terms.

**Comment by damonge** [comment]
@jablazek @shivamp1495 I'm trying to implement this now. My guess is that, in order to get Pk_{b1,b3nl} we need to replace our current call to `one_loop_dd_bias` [here](https://github.com/LSSTDESC/CCL/blob/127b6c4972643780b49419fdba374ee8a2c66a2c/pyccl/nl_pt/power.py#L117), with a call to `one_loop_dd_bias_b3nl` [here](https://github.com/JoeMcEwen/FAST-PT/blob/cf22622c9f80939871df6468e9dda048292aa6a8/fastpt/FASTPT.py#L411).

1. Is that correct?
2. If so, is Pk_{b1, b3nl} given by `sig3nl` in [this line](https://github.com/JoeMcEwen/FAST-PT/blob/cf22622c9f80939871df6468e9dda048292aa6a8/fastpt/FASTPT.py#L451)?

**Comment by shivamp1495** [comment]
Hi David,

Yes both are correct. The example of calling and combining is here https://github.com/JoeMcEwen/FAST-PT/blob/master/examples/fastpt_example_nonlin_bias.py (but unfortunately the b3nl and sig4 terms are swapped in order). Please let me know if I can help in some other way.

Sincerely,
Shivam

**Comment by damonge** [comment]
Thanks a lot @shivamp1495 
Do you mean that there's a "bug" in [this line](https://github.com/JoeMcEwen/FAST-PT/blob/cf22622c9f80939871df6468e9dda048292aa6a8/examples/fastpt_example_nonlin_bias.py#L52) and `sig3nl` and  `sig4kz` should have been swapped?

**Comment by shivamp1495** [comment]
Yes, exactly.

Sincerely,
Shivam

**Comment by jablazek** [comment]
Yeah, it's not really a bug. The FAST-PT function originally had the order swapped, but then we switched them to be consistent with `one_loop_dd_bias`. Never updated the example.

**Comment by damonge** [comment]
OK, great. Thanks a lot both!
I've created a PR (#901) that will eventually solve this issue. I'll ping you both for a review as soon as it's ready.

**Comment by damonge** [comment]
@jablazek , @shivamp1495 I've implemented this in #901 . Could you take a look when you get a chance?
